packages/terra-form/docs/Fieldset.md

```diff
@@ -0,0 +1,23 @@
+# Fieldset
+
+Generic form fieldset component which handles the layout of a standard form fieldset including errors, help text, label, value and widget placement.
+
+## Usage
+
+```jsx
+import React from 'react';
+import Fieldset from 'terra-form/lib/Fieldset';
+
+<Fieldset
+  type="checkbox"
+  legend="Do you have any Children?"
+  name="children_present"
+  value="children_present"
```

Comment:
Are `name` and `value` valid props here? I didn't see it defined in the proptypes for `packages/terra-form/src/Fieldset.jsx`.